lib/ansible/modules/network/nxos/nxos_ospf_vrf.py

```diff
@@ -337,6 +357,9 @@ def state_absent(module, existing, proposed, candidate):
                         command = 'no {0}'.format(key)
                     else:
                         command = None
+                elif key == 'bfd':
```

Comment:
Why not `elif key == bfd and value == 'enable'`?

Reply:
Because if `key == 'bfd'` and `value == 'disable'`, the flow will fall into the the `else:` on line 363 which results in `'no bfd disable'` (invalid cli)  instead of `no bfd`.

Reply:
: